src/ray/core_worker/transport/direct_actor_transport.cc

```diff
@@ -21,8 +21,7 @@ CoreWorkerDirectActorTaskSubmitter::CoreWorkerDirectActorTaskSubmitter(
     : io_service_(io_service),
       gcs_client_(gcs_client),
       client_call_manager_(io_service),
-      store_provider_(
-          object_interface.CreateStoreProvider(StoreProviderType::LOCAL_PLASMA)) {
+      store_provider_(object_interface.CreateStoreProvider(StoreProviderType::MEMORY)) {
```

Comment:
Would prefer to pass the `LocalMemoryStoreProvider` in as an argument to the constructor rather than calling `CreateStoreProvider` from here. This would expose less of the `object_interface` to the transport and make it easier to write unit tests (can just pass in a provider instead of a whole object_interface).

Reply:
OK, thanks, I made the change as suggested.